apps/system/utils/Kconfig

```diff
@@ -12,6 +12,20 @@ config SYSTEM_CMDS
 		system commands. Because they can be used without TASH by function calling.
 		Let's enable system commands at each configuration.
 
+config SECURITY_LEVEL_CMDS
+	bool "Security level commands"
+	default y
+        depends on TASH
```

Comment:
This is being turned on by CONFIG_TASH. It is not TASH default operation.
So If we should set dependency about TASH, How about `depends on TASH && SECURITY_LEVEL`?

Reply:
I'll add it. Thanks.